src/smartpeak/source/ui/ParametersTableWidget.cpp

```diff
@@ -164,6 +181,20 @@ namespace SmartPeak
         }
       }
       ImGui::EndTable();
+      if (edit_parameter)
+      {
+        // we rebuild the Parameter object from the selected row
```

Comment:
The Tensor format was chosen to enable 1) faster plotting and 2) faster data analytics.  None of those two apply here for the parameters so if it makes sense to use a different representation, I would highly recommend that.